lib/ios/Interactable/InteractableView.m

```diff
@@ -286,17 +286,24 @@ - (void)reportAlertEvent
     }
 }
 
+- (void)reportDragEvent:(NSString*)state
+{
+    [self reportDragEvent:state targetSnapPointId:nil];
+}
+
 - (void)reportDragEvent:(NSString*)state targetSnapPointId:(NSString*)targetSnapPointId
 {
     if (self.onDrag)
     {
         CGPoint deltaFromOrigin = [InteractablePoint deltaBetweenPoint:self.center andOrigin:self.origin];
-        self.onDrag(@{
-                        @"state": state,
-                        @"x": @(deltaFromOrigin.x),
-                        @"y": @(deltaFromOrigin.y),
-                        @"targetSnapPointId":targetSnapPointId
-                      });
+        NSDictionary *onDragEvent = [[NSDictionary alloc] initWithObjectsAndKeys:
+                                         state ?: [NSNull null], @"state",
+                                         deltaFromOrigin.x ?: 0, @"x",
+                                         deltaFromOrigin.y ?: 0, @"y",
+                                         targetSnapPointId ?: [NSNull null], @"targetSnapPointId",
```

Comment:
So, assuming that the user hasn't provided an id, `end` returns null instead of an empty string ? was even able to return an empty string before or did it crash ?

Reply:
The crash was based on the circumstance that an Interactable View got no SnapPoints. iOS was crashing then (Getting <id> of not present <point> Object)
I´m glad for discussion if we should either just fix the crash and provide an empty string if no id is provided, or just adapt the "not present" value.
Also i just looked up that i didn´t check Android Start Event. There we are sending an empty string on Start, but an undefined value on end if no <id> is provided.

So we can adapt the previous behaviour and just send out on Android also the empty string
OR
Send out on Start an undefined value, on End undefined if not present or String if present on both platforms

Reply:
Most important is that we align the expected behavior on both platforms. If the Android behavior is working as expected let's just copy it on iOS, empty string is falsy in JS anyway.